services/API-service/src/api/event/trigger-per-lead-time.entity.ts

```diff
@@ -42,8 +42,8 @@ export class TriggerPerLeadTime {
   public eventName: string;
 
   @Column({ default: false })
-  public triggered: boolean;
+  public forecastAlert: boolean;
```

Comment:
Let's add a note to help our future selves to interpret this variable.

```
public forecastAlert: boolean; // true indicates a trigger or warning for this lead time, false indicates neither REFACTOR: to either use this for all disaster types or to remove this exception for floods
```

This looks like it's user-facing via the api so possibly update the wiki to help others also interpret this variable.